content/news/playtest-20150808.md

```diff
@@ -0,0 +1,58 @@
+
+A number of new features, fixes, and performance improvements make this a significant improvement over the last playtest and the June release.
+
+We are happy to announce that we have finally addressed a few long-standing performance issues that caused significant lag spikes in the mid to late game.  As part of this work, the AI has become a lot smarter about base building, uses support powers more reliably, and harvesters are less likely to become stuck.
+
+Music playback has received quite a lot of changes.  Map authors now have much greater control over music, including the ability to specify default (starting) music as well as custom victory and defeat tracks.
+
+Other changes include:
+
+* Spies in the Red Alert mod have received a major overhaul, making them much more useful.
+* Paratroopers can now be selected and given orders before they land.
+* Two new single-player missions have been added, Soviets 04a and 04b.
+* Single-player missions have been polished, and Allies 05 has received a new ultra-hard difficulty.
+* Mission replays can again be viewed through the replay browser.
+* A warning message is now given when a server admin selects a map with custom rules.
+* Range circles are now visible to allied players and spectators.
+* Players can now hide the in-game UI (using a hotkey, bound to Ctrl+Shift+H by default).
+* We now support installing game content from *The First Decade* DVD.
+
+Of interest for modders and mappers will be the addition of new Lua API functions.  Disguising spies, capturing or infiltrating units and buildings using scripting is now possible.  Other newly available functions can be used to play arbitrary sound files, flash the screen and control the new palette effect trait.
+
+Of course we did not only add new things, but also fixed bugs and annoyances.  Here's a small sample:
+
+* A number of crashes have been resolved.
+* The score screen can now be closed to have a look at the battlefield after the match.
+* The 'Options' button will no longer blink for no reason in skirmish maps.
+* The duration bar on chronoshifted units in the Red Alert mod is no longer visible to enemy players.
+* The aircraft and helicopter cruise altitude in the Red Alert mod the has been increased again.
+* The unit duplication crate bonus has been removed from the Tiberian Dawn mod.
+
+For all the gritty details, check out the [full changelog](http://changelog.openra.net).
+
+Grab the installer for this playtest [from our download page](/download/) and give it a try!
+
+<div style="text-align:center" markdown="1">
+
+![Custom map rule warning](/images/news/20150808-cnc-lobby.png)
+
+The game will warn players in the lobby when the host selects a map with custom rules.  This should help combat the small number of dishonest players who have been creating cheat maps (shame on you!).
```

Comment:
I don't like the second sentence here at all.
They aren't *cheat maps* they are maps that alter the rules which we fully allow so it is valid.
That does not make players dishonest either.

I get that it is playful, but I'm not a fan of coming off like this.